internal/pkg/template/templates/workloads/partials/cf/https-listener.yml

```diff
@@ -24,11 +24,11 @@ LoadBalancerDNSAlias:
 {{- range $hostedZoneID, $aliases := .HostedZoneAliases}}
 LoadBalancerDNSAlias{{$hostedZoneID}}:
   Metadata:
-    'aws:copilot:description': 'Alias records for the application load balancer in hosted zone {{$hostedZoneID}}'
+    'aws:copilot:description': 'Alias records for the service in hosted zone {{$hostedZoneID}}'
```

Comment:
:wutface: I think I'm misunderstanding something if there is a `hosted_zone` field specified in the manifest then why are we inserting into the CloudFront hosted zone?

Should the A record be inserted outside of this range: `{{- range $hostedZoneID, $aliases := .HostedZoneAliases}}`? can you explain it to me

Reply:
> why are we inserting into the CloudFront hosted zone
I don't understand the question...we are not inserting into the CF hosted zone. When adding an A-record we need to specify hosted zone for the resource (aka, `PublicLoadBalancerHostedZone` for ALB and each ALB it is different and `Z2FDTNDATAQYW2` for CF the same universally)

>Should the A record be inserted outside of this range: {{- range $hostedZoneID, $aliases := .HostedZoneAliases}}? 
why? An A-record corresponds to an alias.

Reply:
nvm boss I misread 😅 it totally makes sense